package.json

```diff
@@ -57,7 +56,7 @@
     "@types/jest": "^24.9.0",
     "@types/jest-axe": "^3.2.1",
     "@types/node": "12.12.27",
-    "@types/react": "^16.9.22",
+    "@types/react": "16.9.19",
```

Comment:
I'm still seeing type errors related to this package, need to investigate further 🧐